Author: KeePassXC Team <team@keepassxc.org>
Subject: Revert GNU Gettext version to 0.18 to fix compilation errors on old systems

--- libgpg-error-127-1.27.orig/configure
+++ libgpg-error-127-1.27/configure
@@ -12961,7 +12961,7 @@ $as_echo "$USE_NLS" >&6; }
 
 
 
-      GETTEXT_MACRO_VERSION=0.19
+      GETTEXT_MACRO_VERSION=0.18
 
 
 
--- libgpg-error-127-1.27.orig/configure.ac
+++ libgpg-error-127-1.27/configure.ac
@@ -151,7 +151,7 @@ AH_BOTTOM([
 
 
 # Note, that autogen.sh greps for the next line.
-AM_GNU_GETTEXT_VERSION([0.19.3])
+AM_GNU_GETTEXT_VERSION([0.18.3])
 AM_GNU_GETTEXT([external])
 
 # Checks for header files.
--- libgpg-error-127-1.27.orig/m4/po.m4
+++ libgpg-error-127-1.27/m4/po.m4
@@ -1,4 +1,4 @@
-# po.m4 serial 22 (gettext-0.19)
+# po.m4 serial 22 (gettext-0.18)
 dnl Copyright (C) 1995-2014 Free Software Foundation, Inc.
 dnl This file is free software; the Free Software Foundation
 dnl gives unlimited permission to copy and/or distribute it,
@@ -30,7 +30,7 @@ AC_DEFUN([AM_PO_SUBDIRS],
 
   dnl Release version of the gettext macros. This is used to ensure that
   dnl the gettext macros and po/Makefile.in.in are in sync.
-  AC_SUBST([GETTEXT_MACRO_VERSION], [0.19])
+  AC_SUBST([GETTEXT_MACRO_VERSION], [0.18])
 
   dnl Perform the following tests also if --disable-nls has been given,
   dnl because they are needed for "make dist" to work.
--- libgpg-error-127-1.27.orig/po/Makefile.in.in
+++ libgpg-error-127-1.27/po/Makefile.in.in
@@ -8,8 +8,8 @@
 # Please note that the actual code of GNU gettext is covered by the GNU
 # General Public License and is *not* in the public domain.
 #
-# Origin: gettext-0.19
-GETTEXT_MACRO_VERSION = 0.19
+# Origin: gettext-0.18
+GETTEXT_MACRO_VERSION = 0.18
 
 PACKAGE = @PACKAGE@
 VERSION = @VERSION@
