Description: Drop -lgpg-error from libgcrypt-config --libs output.
Author: Andreas Metzler <ametzler@debian.org>
Origin: vendor
Bug-Debian: http://bugs.debian.org/405238
Last-Update: 2014-01-04

--- libgcrypt20-1.6.0.orig/src/libgcrypt-config.in
+++ libgcrypt20-1.6.0/src/libgcrypt-config.in
@@ -159,7 +159,8 @@ if test "$echo_libs" = "yes"; then
     fi
 
     # Set up `libs_final'.
-    libs_final="$libs_final $gpg_error_libs"
+    #libs_final="$libs_final $gpg_error_libs"
+    libs_final="-lgcrypt"
 
     tmp=""
     for i in $libdirs $libs_final; do
