Description: Fix zsh completion generation
Origin: vendor
Forwarded: https://github.com/bagder/curl/pull/582
Author: Alessandro Ghedini <ghedo@debian.org>
Last-Update: 2015-12-27

--- a/scripts/Makefile.am
+++ b/scripts/Makefile.am
@@ -6,7 +6,7 @@
 all-local: $(ZSH_COMPLETION_FUNCTION_FILENAME)
 
 $(ZSH_COMPLETION_FUNCTION_FILENAME): zsh.pl
-	$(PERL) $< > $@
+	$(PERL) $< $(top_builddir)/src/curl > $@
 
 install-data-local:
 	$(MKDIR_P) $(DESTDIR)$(ZSH_FUNCTIONS_DIR)
